app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/HomePresenter.kt

```diff
@@ -39,24 +47,30 @@ class HomePresenter(private val repository: Repository) : HomeContract.Presenter
         })
     }
 
-    override fun getMovie(type: String, page: Int) {
-        repository.getMovies(type, page, object : OnFetchDataJsonListener<MoviesResponse> {
+    override fun getMovie(type: String, page: Int, genresID: Int) {
```

Comment:
```suggestion
    override fun getMovie(
        type: String,
        page: Int = Constant.BASE_PAGE_DEFAULT, 
        genresID: Int = Constant.BASE_GENRES_ID_DEFAULT
    ) {
```
xoá bớt tham số trên onStart()